ivy/functional/backends/torch/device.py

```diff
@@ -87,6 +87,14 @@ def tpu_is_available() -> bool:
     return False
 
 
+def handle_soft_device_variable(*args, fn, dst_dev=None, **kwargs):
+    args, kwargs, dst_dev = ivy.shift_native_arrays_on_def_dev(
```

Comment:
`def_dev` doesn't really convey what the function does, let's rename this to `shift_native_arrays_on_default_device`.